packages/react-i18n/src/index.tsx

```diff
@@ -61,22 +125,92 @@ export const withI18n = createHigherOrderComponent< I18nReact >( ( InnerComponen
 	};
 }, 'withI18n' );
 
+interface MakeContextValueOptions {
+	filters: I18nFilters;
+}
```

Comment:
Instead of creating a wrapper `options` object, maybe we could pass `filters` directly everywhere these `options` are used?